### PR TITLE
nerfs welding shield eyes

### PR DESCRIPTION
**Pull request by TheRyeGuyWhoWillNowDie**
passive immunity to all forms of eye damage and instant win bulbs while being impossible to tell it's there is bad
changes it from passive to active, gives it a welding helmet effect when active
also removes it's emp immunity?

:cl:  
tweak: welding shield eyes have been nerfed
/:cl:

### CODE REVIEW

**Comment by Mqiib** [comment]
is this welding HELMET or welding GOGGLES effect?

File: code/modules/surgery/organs/eyes.dm [hunk 1]
```diff
@@ -242,10 +242,21 @@
 /obj/item/organ/eyes/robotic/shield
 	name = "shielded robotic eyes"
 	desc = "These reactive micro-shields will protect you from welders and flashes without obscuring your vision."
```
Comment (ynot01) [inline code/modules/surgery/organs/eyes.dm]: This is no longer true in your PR

**Review by ynot01 — CHANGES_REQUESTED** [review]

**Comment by Hislyna** [comment]
Sounds like making an item thats decent into a useless item that may as well be removed *shrug

File: code/modules/surgery/organs/eyes.dm [hunk 2]
```diff
@@ -241,11 +241,22 @@
 // Welding shield implant
 /obj/item/organ/eyes/robotic/shield
 	name = "shielded robotic eyes"
-	desc = "These reactive micro-shields will protect you from welders and flashes without obscuring your vision."
-	flash_protect = 2
+	desc = "These reactive micro-shields will protect you from welders and flashes."
```
Comment (ynot01) [inline code/modules/surgery/organs/eyes.dm]: They are not reactive anymore, you have to turn them on yourself

**Review by ynot01 — DISMISSED** [review]

**Comment by RG4ORDR** [comment]
I don't understand.
Can you explain more in depth why? This simply just seems like a nerf to an item that doesn't get a lot of usage.

**Comment by TheRyeGuyWhoWillNowDie** [comment]
> I don't understand. Can you explain more in depth why? This simply just seems like a nerf to an item that doesn't get a lot of usage.

medlizard nerf

**Comment by TheRyeGuyWhoWillNowDie** [comment]
> Sounds like making an item thats decent into a useless item that may as well be removed *shrug

not having to give up your eye or head slot for flash shielding pretty good i think

**Comment by Hislyna** [comment]
> > Sounds like making an item thats decent into a useless item that may as well be removed *shrug
> 
> not having to give up your eye or head slot for flash shielding pretty good i think

Not value when you get screwed from EMP too, this item is barely used as is and will now be used less.

**Comment by TheRyeGuyWhoWillNowDie** [comment]
> this item is barely used as is and will now be used less.

Good

**Comment by Hislyna** [comment]
I feel theres alot better options than this
EMP Protected but activatable
No EMP Protected but passive
Stays as passive but no longer gives flash protection

Even then I feel it doesn't need a nerf in the first place

**Comment by Uristthedorf** [comment]
At this point, why bother with becoming EMP vulnerable and having to get surgery when you can get a welding shield mask from a lathe, which functions just the same but with no downsides.
Seriously welding shield masks are great, no EMP, can eat while wearing it, functions as a gas mask, takes up the mask slot which isn't too valuable, and is fireproof.
Only reason to get nerfed welding shield eyes over welding gas mask is that you can't have it stripped from you.

**Comment by Uristthedorf** [comment]
Changelings and anyone wearing a mask can hide their flash protection with no way to find out besides stripping or flashing them. Why is it bad for welding shield eyes in particular to be hard to see and not anything else?

**Comment by ynot01** [comment]
> Changelings and anyone wearing a mask can hide their flash protection with no way to find out besides stripping or flashing them. Why is it bad for welding shield eyes in particular to be hard to see and not anything else?

Changeling ≠ preternis or surgery-capable metafriend

**Comment by Uristthedorf** [comment]
> > Changelings and anyone wearing a mask can hide their flash protection with no way to find out besides stripping or flashing them. Why is it bad for welding shield eyes in particular to be hard to see and not anything else?
> 
> Changeling ≠ preternis or surgery-capable metafriend

I guess changelings are antag and deserve it, but gas masks (or any other eye covering mask) are an easy way to hide sunglasses.

**Comment by 00ze-cyclone** [comment]
> > I don't understand. Can you explain more in depth why? This simply just seems like a nerf to an item that doesn't get a lot of usage.
> 
> medlizard nerf

Why not just put implants only in robotic if we want to stop that? Only give medbay the organs and robotic are the only ones able to print implants.

**Comment by RG4ORDR** [comment]
Yeah I aint seeing anything to justify this.

Welding shield eyes are aren't used cause sunglasses,welding masks/gasmask,space helmets, and goggles all provide this benefit. All of the above are easy enough to obtain.

**Comment by Bibby0110** [comment]
No need for this nerf. See above comments

**Pull request closed without merging**